config/webhook/manifests.yaml

```diff
@@ -16,7 +16,6 @@ webhooks:
   failurePolicy: Ignore
   matchPolicy: Equivalent
   name: mpod.vpc.k8s.aws
-  timeoutSeconds: 5
```

Comment:
Synced offline- changes to this file should be reverted.

Reply:
I reverted the file.